.clang-tidy

```diff
@@ -0,0 +1,8 @@
+---
+Checks: '-*,
+  modernize-deprecated-headers,
+  '
+
+WarningsAsErrors: '
+#  modernize-deprecated-headers,
```

Comment:
I needed at least one check for clang-tidy to work. The problems found by this check are currently considered only warnings and not errors. The # leaves it commented out in the 'WarningAsErrors' section.